src/clld_markdown_plugin/__init__.py

```diff
@@ -15,36 +16,54 @@
 __version__ = "0.0.1.dev"
 __all__ = ['markdown', 'includeme']
 
-default_model_map = {
-    "LanguageTable": Language,
-    "FormTable": Unit,
-    "ExampleTable": Sentence,
-    "ContributionTable": Contribution,
-    "CognatesetTable": UnitParameter,
-    "ParameterTable": UnitParameter,
-    "sources.bib": Source,
-    "Source": Source,
-}
-model_map = {}
-function_map = {}
+
+def settings(custom: typing.Optional[dict] = None) -> dict:
+    def full_spec(spec):
+        return spec if isinstance(spec, dict) else {
+            'route': spec.__name__.split('.')[-1].lower() if spec else '', 'model': spec}
+
+    res = {
+        'model_map': {
+            "LanguageTable": full_spec(common.Language),
+            "ParameterTable": full_spec(common.Parameter),
```

Comment:
@fmatter mapping `ParameterTable` to `Parameter` by default (rather than `UnitParameter`) seems more reasonable, I think.

Reply:
My reasoning here was based on the assumption that if a CLLD app was to use this plugin, it would be much more likely that the author(s) would want to refer in their markdown content to meanings of forms (which I understand unitparameters to be), rather than typological parameters (which I understand parameters to be).

Reply:
Quite a few of "my" clld apps with lexical data use Parameter for
cross-linguistic word meanings and Value for the forms. The idea here is
that the simplest data model for linguistic data - triples of language,
feature and Value- can also be applied to lexical data like Wordlist.

But I agree, as soon as the forms become objects of study (rather than just
the language), units and unitparameters make more sense.

I'd still vote for the simple case being the default.

Reply:
Fine by me :)